packages/pilot/src/pages/Account/actions/epic.js

```diff
@@ -56,17 +52,7 @@ const isActiveCompany = propEq('status', 'active')
 const isSelfRegister = propEq('type', 'self_register')
 const isPendingRiskAnalysis = propEq('status', 'pending_risk_analysis')
 
-const pathNotEq = (pathName, propValue) => complement(
-  pathEq(pathName, propValue)
-)
-
-const hasDashboardAccess = ifElse(
-  hasPath(['client', 'authentication', 'allow_dashboard_login']),
-  path(['client', 'authentication', 'allow_dashboard_login']),
-  allPass([
-    pathNotEq(['company', 'type'], 'mei'),
-  ])
-)
+const hasDashboardAccess = company => path(['company', 'capabilities', 'allow_dashboard_login'], company)
```

Comment:
só ajustar a função pra puxar a capability da company:

```suggestion
const hasDashboardAccess = path(['capabilities', 'allow_dashboard_login'])
```